docs/concepts/faq.md

```diff
@@ -34,6 +35,20 @@ When you add a file to IPFS, IPFS splits it into smaller blocks. IPFS hashes eac
 
 There are a lot of ways you can contribute to IPFS, whether you're interested in helping with either of the core implementations, applications like IPFS Desktop, writing or editing documentation, doing UX, or whatever you enjoy working on. [Get all the details on where to get started here.](../community/contribute/ways-to-contribute.md)
 
+### What is an implementation
```

Comment:
We should really be driving people towards https://specs.ipfs.tech in this section and particularly https://specs.ipfs.tech/architecture/